test/recipes/30-test_evp_data/evprand.txt

```diff
@@ -79795,6 +79795,17 @@ Result = EVP_RAND_CTX_set_params
 
 Title = Test FIPS indicator callbacks for truncated digests
 
+FIPSversion = >=3.4.0
```

Comment:
These ones are different from the rest :)

Reply:
In what sense?

Reply:
no available in?

Reply:
These are new tests.. all good..  The others are just adding a reason

Reply:
Yeah, I noticed the lack.  I've added available in lines too.  They'll be required eventually.